rasa/core/nlg/interpolator.py

```diff
@@ -0,0 +1,23 @@
+import re
+
+
+def interpolate_text(text, values):
```

Comment:
I'm not so happy with the name. Maybe something like `transforming_text_template` and `transforming_custom_template`?

Reply:
sure thing, happy to update, got the name from 'string interpolation' (https://en.wikipedia.org/wiki/String_interpolation)

Reply:
Did not know about that. Then we should maybe keep it. Sorry.

Reply:
All good! Thanks for the feedback!